tests/create.pp

```diff
@@ -83,3 +83,19 @@
     port     => 80,
   }],
 }
+
+rds_instance { 'db-name-5':
```

Comment:
Instead of placing these in the simple test file could you create an example in the examples directory? See the other examples for an, er, example.